lib/src/layout_adjustments/frame.dart

```diff
@@ -1,17 +1,33 @@
 import 'package:flutter/widgets.dart';
 
+import 'alignment.dart' as swift_ui;
+
+/// A layout widget that aligns its child within a fixed-size rectangle.
 class Frame extends StatelessWidget {
   const Frame({
     super.key,
     this.width,
     this.height,
-    this.alignment = Alignment.center,
+    this.alignment = swift_ui.Alignment.center,
```

Comment:
I was trying to do that but still needed `Alignment` from the widgets library to pass to the Align widget. I think it might work if I follow your other suggestion to make a `toFlutter` method.